dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+                        .append("Object.getFriendlyName()}) )");
+
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("ImagePath=$!{")
+                        .append(field.variable())
+                        .append("Object.getPath()})");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("ImageName=$!{")
+                        .append(field.variable())
+                        .append("Object.getFileName()})");
+
+                } else {
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileInode=$!{")
+                        .append(field.variable())
+                        .append("Object.getInode()} )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileIdentifier=$!{")
+                        .append(field.variable())
+                        .append("Object.getIdentifier()} )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileExtension=$!{")
+                        .append(field.variable())
+                        .append("Object.getExtension()} )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileURI=$filetool.getURI($!{")
+                        .append(field.variable())
+                        .append("Object}, ")
+                        .append(content.getLanguageId())
+                        .append(" ))");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileTitle=$!{")
+                        .append(field.variable())
+                        .append("Object.getTitle()} )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileFriendlyName=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getFriendlyName()} ))");
+
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FilePath=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getPath()}) )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileName=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getFileName()} ))");
+                }
+            } // http://jira.dotmarketing.net/browse/DOTCMS-2178
+            else if (field instanceof BinaryField) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)